spec/defines/dns_zone_spec.rb

```diff
@@ -38,7 +38,7 @@
       :group    => 'named',
       :mode     => '0644',
       :replace  => 'false',
-      :notify   => 'Service[named]',
+      :notify   => 'Class[Dns::Service]',
```

Comment:
A more correct way of checking uses the catalog. I realize this was already incorrect, but I'd appreciate it if you fixed it while changing this part
```suggestion
      ).that_notifies('Class[Dns::Service]')
```